debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -2860,6 +2860,39 @@ public void shouldOutputRecordsInCloudEventsFormat() throws Exception {
         }
     }
 
+    // YB Note: Running this test also requires a change of values in the method TestHelper#defaultJdbcConfig
```

Comment:
Could add a comment here that this test is for manual testing.
Also instructions on what to do and when